src/sources/AM32/eeprom.js

```diff
@@ -34,6 +34,10 @@ const LAYOUT = {
     offset: 0x05,
     size: 12,
   },
+  TYPE: {
```

Comment:
Please move to the end of the end of the layout - the values here are sorted by offset.